src/FWF.jl

```diff
@@ -19,7 +19,9 @@ Configuration Settings for fixed width file parsing.
   * `skip`        : integer of the number of lines to skip; default `0`
   * `trimstrings` : true if strings should be trimmed; default `true`
   * `usemissings` : true if fields should be checked for null values; default `true`
-  * `skiponerror` : true if errors should not throw an exception; default `true`
+  * `errorlevel`  : if `:parse` then as much as possible is parsed and missing data is replaced by `missing`;
```

Comment:
For this an the associated code changes.  We should in some way not let the user create a case where `usemissings == false && errorlevel == :parse`  Because I think my code only unions column types with missing if `usemissings == true`.

Reply:
I thought this is what was intended, i.e. if `usemissings == false` then having `missing` anywhere throws an error.

I can add check that makes sure that if `errorlevel == :parse` then `usemissings` must be `true`.

Reply:
You made me think and an exception probably isn’t the proper approach nor is forcing parameters.  

Someone could have usemissings == false and errorlevel == :parse and never get an error if their file doesn’t have malformed lines.

But to inform the user I think we should write out a warning message if usemissings == false and errorlevel == :parse as that combo could lead to unexpected exceptions.

Reply:
fixed